src/main.ts

```diff
@@ -32,17 +32,34 @@ async function run() {
     return;
   }
 
-  const hasA11yLabel = issueOrPullRequestHasLabel("A11Y");
-  if (!hasA11yLabel && !issue_title.includes('[A11Y]')) {
-    console.log('Not an accessibility issue, skipping.')
-    return;
-  }
-
+  const LABELS = {
+    A11Y: getRequiredInput('label-a11y'),
```

Comment:
Only thing that looks a little odd is the `getRequiredInput` function having `required: true` when the inputs do have a default defined in `action.yml`. It does [appear the behavior of `@actions/core`](https://github.com/actions/toolkit/blob/74906bea83a0dbf6aaba2d00b732deb0c3aefd2d/packages/core/src/core.ts#L69-L77) should not throw because of the default value so this is probably okay.

Reply:
Good eye. I had some same thought. The way I see it, from the perspective of the action script the input is indeed required, because it is necessary for the script to work properly. Someone could break the contract by altering the action.yml configuration and removing the default value, or the input entirely, in which case a helpful error would be produced.

If instead you think of "required" as meaning that the user must provide a value, then I'm definitely misusing the concept there.

I'd be interested to know what you think.

Reply:
Yeah it threw me off because the function name `getRequiredInput` made me think the input would be required as in provided via the user, when that's not the case. It's all semantics and I think it's fine since it is technically required that a value exists so it should throw if the user does something weird like `label-a11y: false` in their own action.